### DISCUSSION TITLE
Yolov8 models with 1 channel images (gray images)

### DISCUSSION THREAD

**Discussion by bzisl** [ultralytics/ultralytics#803] [Q&A]
I want to know of someone has been able to use only gray images and to reduce the input size from 3 channels to 1 channel.
For small devices is quite interesting by two reasons: images occupy three times less and CPU doesn't require to expand 1 channel image to RGB.
Any idea of how to do it?

**Comment by glenn-jocher** [comment]
@bzisl hello! Yes, YOLOv8 can be adapted to work with 1-channel grayscale images instead of the standard 3-channel RGB images. To do this, you would need to modify the model's configuration to accept single-channel input. This involves adjusting the input layer to accept a single channel and ensuring that any data preprocessing steps are consistent with this change, such as loading images as grayscale during both training and inference.

Keep in mind that this modification requires a solid understanding of the model architecture and the data pipeline. Also, be aware that training from scratch with grayscale images might be necessary, as the pre-trained weights are typically optimized for 3-channel RGB images.

For more detailed guidance on model configuration and training, you can refer to our documentation at https://docs.ultralytics.com. Good luck with your project! 😊🚀
